templates/security/email/change_notice.html

```diff
@@ -0,0 +1,60 @@
+<style>
+div {
+    margin: auto;
+    width: 50%;
+    background-color: #EAECEE;
+}
+table {
+    margin: auto;
+    text-align: center;
+    font-family: 'Lato', Helvetica, Arial, sans-serif;
+}
+
+.reset_button {
+    border-radius: 5px;
+    background-color: #3498DB;
+    border: none;
+    font-size: 15px;
+    text-decoration: none;
+    color: white;
+    padding: 10px 24px;
+}
```

Comment:
for which email clients was the styling tested?

Reply:
@kpsherva on the fake smtp server [Mailhog](https://github.com/mailhog/MailHog) only